javascript/src/constants.js

```diff
@@ -1,16 +1,12 @@
 const {
   TRANSCEND_API_KEY,
-  ENRICHMENT_SIGNING_KEY,
-  ORGANIZATION_URI,
```

Comment:
any merit in checking `audience` on `jwt.verify`?

Reply:
ya potentially, but its one more variable for them to manage and noticed you removed it in the js code. happy to do either way

Reply:
yeah it's safe w/o